### PR TITLE
Fix news details DTO

### PR DESCRIPTION
**Pull request by Dreylak**
- Added isLikedByCurrentUser property in news details
- Created TryGetAuthUserAsync method that return null instead of throwing exception if user not found

### CODE REVIEW

File: TheraLang.BLL/Services/NewsService.cs [hunk 1]
```diff
@@ -81,14 +83,18 @@ public async Task<NewsDetailsDto> GetNewsById(int id)
...
             {
                 cfg.CreateMap<News, NewsDetailsDto>()
                     .ForMember(m => m.ContentImageUrls,
                         opt => opt.MapFrom(sm => sm.UploadedContentImages.Select(i => i.Url)))
                     .ForMember(m => m.AuthorName,
                         opt => opt.MapFrom(sm => $"{sm.Author.Details.FirstName} {sm.Author.Details.LastName}"))
-                    .ForMember(m => m.LikesCount, opt => opt.MapFrom(sm => sm.UsersThatLiked.Count));
+                    .ForMember(m => m.LikesCount, opt => opt.MapFrom(sm => sm.UsersThatLiked.Count))
+                    .ForMember(m => m.IsLikedByCurrentUser,
+                        opt => opt.MapFrom(sm => sm.UsersThatLiked.Select(u => u.Id).Contains(currentUser.Id)));
```
Comment (oleksiistetsiouk) [inline TheraLang.BLL/Services/NewsService.cs]: will an error be thrown here if currentUser is null?

Comment (oleksiistetsiouk) [inline TheraLang.BLL/Services/NewsService.cs]: will an error be thrown here if currentUser is null?

Comment (oleksiistetsiouk) [inline TheraLang.BLL/Services/NewsService.cs]: will an error be thrown here if currentUser is null?

File: TheraLang.BLL/Services/AuthenticationService.cs [hunk 2]
```diff
@@ -71,5 +71,28 @@ public async Task<AuthUser> GetAuthUserAsync()
                 };
             });
         }
+
+        public async Task<AuthUser> TryGetAuthUserAsync()
```
Comment (ivanko-yuz) [inline TheraLang.BLL/Services/AuthenticationService.cs]: it's either not good naming or we should refactor this method to match TRY-GET pattern ->  bool TryGetValue(id, our value) signature
https://ayoungdeveloper.com/post/2017-03-28-using-the-tryget-pattern-in-csharp/

**Pull request merged**